### DISCUSSION TITLE
求助求助

### DISCUSSION THREAD

**Discussion by Anonymous755** [share121/inter-knot#639] [其他]
![Uploading img498612626_1722692308859.jpg…]()

各位熟悉绳网的大神教教我怎么换头像？

**Comment by FrostPhoenix79** [comment]
点击左上角的头像，点击Public profile，往下翻，翻到name上面有个大圈框起来你的头像，然后左下角有edit标志的，点击就可以替换了

**Comment by Griffin233** [comment]
这个换头像去github换的
